Gemfile

```diff
@@ -11,6 +11,7 @@ group :test do
   gem 'metadata-json-lint', '~> 0.0'
   gem 'rspec-puppet-facts', '~> 1.3'
   gem 'semantic_puppet', '0.1.3'
+  gem 'puppet-blacksmith', '>= 3.4.0',      :require => false, :platforms => 'ruby'
```

Comment:
See https://travis-ci.org/puppetlabs/puppetlabs-puppet_agent/builds/176248555. We should use a PR for merge-up to master to ensure that gets fixed.